modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlSynchronousApiTest.java

```diff
@@ -93,6 +97,31 @@ protected IgniteSql igniteSql() {
         return CLUSTER_NODES.get(0).sql();
     }
 
+    /**
+     * Checks that schema version is updated even if column names are intersected.
+     */
+    @Test
+    public void checkSchemaUpdatedWithEqAlterColumn() {
```

Comment:
We need to stop add tests to random classes. ItSqlSynchronousApiTest is about validating sql API, not integration of DdlCommandHandler and TableManager